app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,8 @@
 class SessionsController < ApplicationController
+  def in
+    head :ok
```

Comment:
I did `head :ok` here so that when the page loads in an iframe it doesn't load any other resources.  I tried sandboxing the iframe originally but that caused warnings to be thrown in the console when the JS couldn't run and this seemed like a cleaner option.